pkg/mhooks/event_type.go

```diff
@@ -29,6 +29,7 @@ const (
 	EventTypeRepresentativeCreated    EventType = "representative.created"
 	EventTypeRepresentativeDeleted    EventType = "representative.deleted"
 	EventTypeRepresentativeUpdated    EventType = "representative.updated"
+	EventTypeTest                     EventType = "event.test"
```

Comment:
I'm considering naming this `EventTypeTestPing` to match method call of `event.TestPing()`.. but const value here is just `event.test`

Reply:
why not `EventTypePing`? think "ping" on its own is pretty universally implied as a test

Reply:
mm i'd prefer to still include `Test` in the name to match the value